server/server.ts

```diff
@@ -112,21 +111,16 @@ async function init() {
   server.listen(process.env.PORT || 8080);
 }
 
-if (process.env.YOUTUBE_API_KEY) {
-  Youtube.authenticate({
-    type: 'key',
-    key: process.env.YOUTUBE_API_KEY,
-  });
-}
+authenticateYoutube();
```

Comment:
I would just move this call to the youtube code file itself. I don't think it makes sense to use the API without authenticating it.